src/applications/vaos/appointment-list/pages/PastAppointmentsPage/index.jsx

```diff
@@ -174,6 +166,18 @@ export default function PastAppointmentsPage() {
         {dateRangeOptions[pastSelectedIndex]?.label}
       </div>
 
+      <div className="vads-u-margin-top--neg1 vads-u-margin-bottom--2">
+        {/* The following line tag is for italics not an icon */}
+        {/* eslint-disable-next-line @department-of-veterans-affairs/prefer-icon-component */}
+        <i>
+          Showing {pastAppointments.length} appointments between today and{' '}
+          {format(
+            dateRangeOptions[pastSelectedIndex].startDateRaw,
+            'MMMM d, yyyy',
+          )}
+        </i>
```

Comment:
Instead of using the `<i></i>` tag, you can add the className “vads-u-font-style--italic” to the div tag